ee/tabby-db/migrations/0029_merged-provider-tables.up.sql

```diff
@@ -11,6 +11,7 @@ CREATE TABLE integration_access_tokens(
 CREATE TABLE provided_repositories(
     id INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
     integration_access_token_id INTEGER NOT NULL,
+    kind TEXT NOT NULL,
```

Comment:
do not duplicate information, use `ntegration_access_token.kind`.

Reply:
Since the pagination query is generated, I'm not sure how we want to do a join there, so if we don't store it in the row we're losing the ability to filter on it